Local logout-menu defaults, kept from the vendored copy: i3lock on a black
background, Hibernate instead of Suspend, and Power off runs reboot.

--- a/logout-menu-widget/logout-menu.lua
+++ b/logout-menu-widget/logout-menu.lua
@@ -52,16 +52,16 @@
     local font = args.font or beautiful.font
 
     local onlogout = args.onlogout or function () awesome.quit() end
-    local onlock = args.onlock or function() awful.spawn.with_shell("i3lock") end
+    local onlock = args.onlock or function() awful.spawn.with_shell("i3lock -n -c 000000") end
     local onreboot = args.onreboot or function() awful.spawn.with_shell("reboot") end
-    local onsuspend = args.onsuspend or function() awful.spawn.with_shell("systemctl suspend") end
-    local onpoweroff = args.onpoweroff or function() awful.spawn.with_shell("shutdown now") end
+    local onsuspend = args.onsuspend or function() awful.spawn.with_shell("systemctl hibernate") end
+    local onpoweroff = args.onpoweroff or function() awful.spawn.with_shell("reboot") end
 
     local menu_items = {
         { name = 'Log out', icon_name = 'log-out.svg', command = onlogout },
         { name = 'Lock', icon_name = 'lock.svg', command = onlock },
         { name = 'Reboot', icon_name = 'refresh-cw.svg', command = onreboot },
-        { name = 'Suspend', icon_name = 'moon.svg', command = onsuspend },
+        { name = 'Hibernate', icon_name = 'moon.svg', command = onsuspend },
         { name = 'Power off', icon_name = 'power.svg', command = onpoweroff },
     }
 
